core/storage/topic/gae_models_test.py

```diff
@@ -136,7 +151,7 @@ def test_has_reference_to_user_id(self):
             topic_models.TopicCommitLogEntryModel
             .has_reference_to_user_id('x_id'))
 
-    def test__get_instance_id(self):
+    def test__get_instance_id(self) -> None:
```

Comment:
This test name is a bit confusing since it tests a different thing, can we rename it while we are here.

Reply:
Here in a comment it is mentioned that:
```
 # Calling create() method calls _get_instance (a protected method)
# and sets the instance id equal to the result of calling that method.
```

So changing it to test__get_instance. Thanks!